core-common/src/main/java/org/glassfish/jersey/logging/ClientLoggingFilter.java

```diff
@@ -64,6 +64,7 @@
      *                      logging filter will print (and buffer in memory) only the specified number of bytes
      *                      and print "...more..." string at the end. Negative values are interpreted as zero.
      *  separator      delimiter for particular log lines. Default is Linux new line delimiter
+     *  redactHeaders  a collection of HTTP headers to be redacted when logging.
```

Comment:
The builder already has a `Collection<String>`. The parsing happens here, right after reading the properties:
https://github.com/eclipse-ee4j/jersey/blob/4a0110070490d590f86d2a1b5809da0dac8cad3e/core-common/src/main/java/org/glassfish/jersey/logging/LoggingFeature.java#L339-L346
https://github.com/eclipse-ee4j/jersey/blob/4a0110070490d590f86d2a1b5809da0dac8cad3e/core-common/src/main/java/org/glassfish/jersey/logging/LoggingFeature.java#L356-L357

Do you think this is reasonable? Should the builder just pass along the string and then have `LoggingInterceptor` deal with the parsing in the constructor?
I went this way because `Verbosity` and `Level` are also parsed there.

Reply:
@nunomsantos, those notes by @jansupol were related only to text modification of JavaDocs. Not to changes in logic. 
In the JavaDoc, you have ``a collection of HTTP headers to be redacted when logging.``. It was requested to change this text to ``a semicolon-separated collection of HTTP headers to be redacted when logging.`` that's it. 

The same is valid for note at line 67 / Add The headers are semicolon-separated. / means just to add that sentence to the existing JavaDoc. 

Thank you.

Reply:
I understand that. For the other comment I updated the Javadoc.
But for this one I would like to clarify that the change would make the Javadoc incorrect because it refers to the builder, which already contains a parsed collection of strings (and not a single string semicolon-separated).

Reply:
Indeed, and as soon as both ``LOGGING_FEATURE_REDACT_HEADERS_SERVER`` and ``LOGGING_FEATURE_REDACT_HEADERS_CLIENT``  properties are documented correctly, I would say that the original version of the JavaDoc is proper as well (without semicolumn I mean). So, @jansupol, could you please confirm that?

Reply:
Those I do think should be changed (see 4a01100) because when the property is loaded it expects a semicolon-separated string.

Reply:
@nunomsantos but you've added the phrase ``The headers are semicolon-separated.`` there. So, it's correct, what I'm saying.